src/wizard/ValidatableWizardStep.vue

```diff
@@ -49,7 +55,10 @@ export default class ValidatableWizardStep<TModel> extends Validatable {
       if (this.skipValidation) return;
 
       this.incomingModel = JSON.parse(JSON.stringify(this.model)) as TModel;
-      this.touched = await this.$store.dispatch("isStepTouched", this.step);
+      this.touched = await this.$store.dispatch(
+        "wizard/isStepTouched",
```

Comment:
I know this was just a refactor for /wizard, but `isStepTouched` is a getter, not an action. - so it's always undefined, which throws an error and makes `this.touched` undefined, so doesn't validate on line 63.

Reply:
fixed this... I caught it earlier.